ui/src/pages/ViewEvent.jsx

```diff
@@ -33,37 +31,30 @@ class ViewEvent extends React.Component {
     );
   }
 
-  showAddPage() {
-    this.setState({showAddPage: true});
-  }
-
-  removeAddPage() {
-    this.setState({showAddPage: false});
-  }
+  showAddPage = () => this.setState({ showAddPage: true });
 
-  render() {
-    const event = this.props.event;
+  removeAddPage = () => this.setState({ showAddPage: false });
 
-    const addPage = (
+  getAddPage(){
+    return (
       <div className="container">
       <AddToTrip event={this.props.event}/>
-      <Button grey label="Cancel" onClick={() => {this.removeAddPage()}} className="cancel-button"/>
+      <Button grey label="Cancel" onClick={() => {this.removeAddPage}} className="cancel-button"/>
```

Comment:
I don't think this works? It doesn't actually call `this.removeAddPage`. It needs to be `onClick={this.removeAddPage}`

Reply:
oops lol, i'll fix that real quick